src/eventtypes.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+
+const inputElement = (e: EventTarget) =>
```

Comment:
use case: controlling a text input on a react component by intercepting the event and using `event.target.value`.

flow can't know what type `event.target` will be, since `KeyboardEvents` can be triggered from any `HTMLElement`. this is a type coercion method to let flow know no, really, `typeof event.target === 'HTMLInputElement'`.